### ISSUE TITLE
[Request] Add support for OpenHardwareMonitor

### ISSUE THREAD

**Issue by pmcanseco** [MarcoPixel/SysDash#18]
It would be awesome if we could get PC hardware monitoring from an already running instance of [Open Hardware Monitor](http://openhardwaremonitor.org/) in order to avoid having more programs running in the background if you don't already use it. They expose all the information via a built in configurable webserver that serves a json file.

**Comment by butlawr** [comment]
I agree this would be great as i reported already on the core temps not right for SpeedFan its not great.

**Comment by Silver2410** [comment]
+1

[HWiNFO](https://www.hwinfo.com) is another alternative.
